packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -105,3 +118,31 @@ function buildCQLinks(pdResults: OutboundPatientDiscoveryResp[]): CQLink[] {
     };
   });
 }
+
+async function augmentPatientDemographics(
+  patient: Pick<Patient, "id" | "cxId">,
+  pdResults: OutboundPatientDiscoveryResp[]
+): Promise<boolean> {
+  const patientResources = fetchPatientResources(pdResults);
+  const successfulAugmentations = await Promise.all(
+    patientResources.map(async patientResource => {
+      return await testAndUpdatePatientDemographics({
+        patient,
+        patientResource,
+      });
+    })
+  );
+  return successfulAugmentations.some(isAugmented => isAugmented === true);
+}
+
+function fetchPatientResources(
```

Comment:
nit: "fetch" gives the idea of retrieving from somewhere remote. 😄  Maybe "get" instead?

Reply:
Done.